incubating/healthcheck/step.yaml

```diff
@@ -74,17 +84,25 @@ spec:
         "additionalProperties": false,
         "patterns": [],
         "required": [
-            "TYPES",
-            "CLUSTER",
-            "NAMESPACE",
-            "DEPLOYMENT",
-            "KUBE_CONFIG"
+            "TYPES"
         ],
         "properties": {
             "CLUSTER": {
                 "type": "string",
                 "description": "Kubernetes Context Name"
             },
+            "DATADOG_API_KEY": {
+                "type": "string",
+                "description": "Datadog API Key."
+            },
+            "DATADOG_APP_KEY": {
+                "type": "string",
+                "description": "Datadog App Key."
+            },
+            "DATADOG_SLO_LIST": {
+                "type": "string",
+                "description": "String of SLO Names delimited with semi-colon character.  Ex. slo-1;slo-2"
```

Comment:
This isn't really necessary but I like to give the option to have the user specify the delimiter (can help with weird naming schemes and reserved characters at times)

Reply:
I do like that option.  I may introduce it later but for now sticking with this delimiter used in the types as well.